### PR TITLE
DE Update 1.0

### PR DESCRIPTION
**Pull request by zRxnx**

### CODE REVIEW

File: language/de.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [{
-    "translator": "EasyAdmin",
+    "translator": "zRxnx",
```
Comment (Blumlaut) [inline language/de.json:2]: i dont think you did this translation entirely on your own, change to `EasyAdmin, zRxnx` at least, please.

File: language/de.json [hunk 2]
```diff
@@ -228,8 +229,8 @@
     "adminimmune": "Dazu hast du keine Rechte, der ausgewählte Spieler ist Immun",
     "teleportmeback": "Teleportiere mich zurück",
     "teleportplayerback": "Teleportiere Spieler zurück",
-    "copiedtoclipboard":"Text wurde in die Zwischenablage kopiert!",
+    "copiedtoclipboard": "Text wurde in die Zwischenablage kopiert!",
 
-    "spectatevehicleseatoccupied":"The vehicle seat you were in is now occupied.",
-    "spectatenovehiclefound":"The vehicle you were in can no longer be found."
+    "spectatevehicleseatoccupied": "Der Fahrzeugsitz auf dem du gessesen bist, ist jetzt besetzt",
```
Comment (Blumlaut) [inline language/de.json:234]: would `hast` fit besser here?

File: language/de.json [hunk 3]
```diff
@@ -228,8 +229,8 @@
...
+    "spectatenovehiclefound": "Das Fahrzeug in dem du gessesen bist, wurde nicht mehr gefunden"
```
Comment (Blumlaut) [inline language/de.json:235]: same as above

**Review by Blumlaut — CHANGES_REQUESTED** [review]

File: language/de.json (hunk 2 above)
Comment (zRxnx) [reply language/de.json]: Overall yes, but at my place is "bist" more used because of that i used "bist".

File: language/de.json (hunk 1 above)
Comment (zRxnx) [reply language/de.json]: Right, sry!

**Pull request merged**